packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+{
+	"manufacturer": "Ecolink",
+	"manufacturerId": "0x014a",
+	"label": "TBH500",
+	"description": "Z-Wave Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x5442",
+			"productId": "0x5439"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "System Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Standard",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Fan Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Gas (No fan w/Heat)",
+					"value": 0
+				},
+				{
+					"label": "Electric (Fan w/Heat)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Change Over Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "w/Cool",
+					"value": 0
+				},
+				{
+					"label": "w/Heat",
+					"value": 1
+				}
+			]
```

Comment:
These labels seem weird. Can you turn them into something more natural sounding?
I'm also not sure what this parameter does. Maybe it needs a better combination of parameter and option labels.

In general we think it is okay to modify the manufacturer's label to something clearer. They are often pretty bad.

Reply:
Has to do with Heat Pumps and how the device is wired. Per the manual: 

**• Changeover type (For Heat Pump HVAC systems only)**
The changeover (or reversing) valve is used to change from
heating to cooling operation. It is either a changeover with
cooling type (Orange wire) or changeover with heating
type (Brown wire). Most are changeover with cooling, which
is the default setting.
- For Changeover with Cooling systems (Orange wire),
select “WITH COOL”. This is the default setting.
- For Changeover with Heating systems (Brown wire),
use the Up/Dn arrows to change to “WITH HEAT”.

I can add the last two bullets as option labels, and add a description that includes bits of the stuff following the first bullet if that makes sense.

Reply:
I think the following is enough, what do you think?

label: "Changeover Type"
description: "For Heat Pump HVAC systems only"
options: "Changeover with Cooling systems (Orange wire)" and "Changeover with Heating systems (Brown wire)".

I don't mind repeating the "Changeover" in the option labels here.

Reply:
Didn't realize you could do Options like that. I think what you have is fine. Will change that.

Reply:
I like it. Will change.